flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/BootstrapTools.java

```diff
@@ -610,4 +613,32 @@ public Config getAkkaConfig() {
 			return AkkaUtils.getThreadPoolExecutorConfig(this);
 		}
 	}
+
+	/**
+	 * Get dynamic properties based on two Flink configuration. If base config does not contain and target config
+	 * contains the key or the value is different, it should be added to results. Otherwise, if the base config contains
+	 * and target config does not contain the key, it will be ignored.
+	 * @param baseConfig The base configuration.
+	 * @param targetConfig The target configuration.
+	 * @return Dynamic properties as string, separated by space.
+	 */
+	public static String getDynamicProperties(
+		org.apache.flink.configuration.Configuration baseConfig,
```

Comment:
Better to import `org.apache.flink.configuration.Configuration` package in class, then we do not need to split lines here.

Reply:
Make sense. I will fix it.